services/streammanager/kafka/client/client.go

```diff
@@ -64,6 +66,37 @@ func New(network string, addresses []string, conf Config) (*Client, error) {
 		}
 	}
 
+	if conf.SSHConfig != nil {
+		signer, err := ssh.ParsePrivateKey([]byte(conf.SSHConfig.PrivateKey))
+		if err != nil {
+			return nil, fmt.Errorf("cannot parse SSH private key: %w", err)
+		}
+
+		sshConfig := &ssh.ClientConfig{
+			User:    conf.SSHConfig.User,
+			Auth:    []ssh.AuthMethod{ssh.PublicKeys(signer)},
+			Timeout: conf.DialTimeout,
+		}
+		if conf.SSHConfig.AcceptAnyHostKey {
+			sshConfig.HostKeyCallback = ssh.InsecureIgnoreHostKey() // skipcq: GSC-G106
+		}
+
+		dialer.DialFunc = func(ctx context.Context, network, address string) (net.Conn, error) {
+			sshClient, err := ssh.Dial("tcp", conf.SSHConfig.Host, sshConfig)
```

Comment:
Why do we create sshClient for every connection? Why not use single client and multiplexing connections?

Reply:
Never mind. Maybe one broker not responding can hand the central ssh connection. And downside is not much as we create as many ssh clients as brokers, which is not very high.